cmd/githubrunnerscalesetlistener/main.go

```diff
@@ -155,3 +158,38 @@ func validateConfig(config *RunnerScaleSetListenerConfig) error {
 
 	return nil
 }
+
+func actionsClientFromConfig(config RunnerScaleSetListenerConfig, creds *actions.ActionsAuth, options ...actions.ClientOption) (*actions.Client, error) {
```

Comment:
Since this is always building a new actions client, can we name it like `newActionsClientFromConfig`?